src/components/ConfigValidation/ConfigIndicator.tsx

```diff
@@ -57,7 +57,9 @@ export class ConfigIndicator extends React.PureComponent<Props, {}> {
   numberOfChecks = (type: string) => {
     let numCheck = 0;
     this.props.validations.forEach(validation => {
-      numCheck += validation.checks.filter(i => i.severity === type).length;
+      if (validation !== undefined) {
```

Comment:
I wonder whether
```
if (validation) { ... }
```
But I wonder if the mapping of `this.props.validations` cannot force to have non undefined values.

Reply:
With backend change always returning empty one, this check isn't needed anymore.